netlify.toml

```diff
@@ -1,4 +1,4 @@
 [build]
   base    = "v2-website"
   publish = "v2-website/build"
-  command = "sed -i -e \"s|const baseUrl .*|const baseUrl = '/'|g\" docusaurus.config.js && yarn install && yarn sync-community-repos && yarn build"
```

Comment:
yarn sync-community-repos is needed to get the list of community repos, it's essential I believe.

Reply:
yup, I was trying to see which part was causing the problem. I think I got it

Reply:
guess not! 😅 I'm still missing something, I'll try again tmr
Anyway, I deployed a test site using the same `netlify.toml` (without the `yarn sync-community-repos`) here:
repo: https://github.com/teikjun/react-native-deploy
test site (netlify): https://react-native-v2.netlify.app/
test site (vercel): https://react-native-deploy.vercel.app/ (same thing)

Reply:
Hey @teikjun, were you able to get the deploy preview working finally? Can I help?

Also, can you put the preview links at the top so that they are more easy to find ? :)

Reply:
I couldn't figure out what was causing the deploy preview to fail, so we're using our own previews for now. 
@darshkpatel has just added the preview links to his PR description at the top! :)